test/service/azure_service_test.rb

```diff
@@ -0,0 +1,69 @@
+require "service/shared_service_tests"
+require "httparty"
+require "uri"
+
+# FIXME: Azure gem can't receive StringIO as body
+if SERVICE_CONFIGURATIONS[:azure]
```

Comment:
Will make this work with the `SharedServiceTest` in the finished PR. Azure storage gem is not working when passing `StringIO` as a body. This is just so I can test at the moment :)